briefing/models.py

```
@@ -0,0 +1,47 @@
+    ])
+    return hashlib.sha256(raw.encode()).hexdigest()[:16]
+
+
+class BriefingSummaryCache(models.Model):
+    briefing = models.ForeignKey(Briefing, on_delete=models.CASCADE, related_name="summary_caches")
+    profile_signature = models.CharField(max_length=16, db_index=True)
+    generated_summary = models.JSONField(help_text="AI가 생성한 요약 (불릿 리스트)")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ["briefing", "profile_signature"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)